gtk/src/entry.rs

```diff
@@ -0,0 +1,21 @@
+use glib::object::IsA;
+use glib::translate::ToGlibPtr;
+use std::convert::TryFrom;
+
+use Entry;
+
+pub trait EntryExtManual: 'static {
+    fn get_invisible_char(&self) -> Option<char>;
+}
+
+impl<O: IsA<Entry>> EntryExtManual for O {
+    fn get_invisible_char(&self) -> Option<char> {
+        let ret = unsafe { gtk_sys::gtk_entry_get_invisible_char(self.as_ref().to_glib_none().0) };
+
+        if ret == 0 {
+            return None;
+        }
+
+        Some(TryFrom::try_from(ret).unwrap())
```

Comment:
```suggestion
        Some(TryFrom::try_from(ret).expect("blabla"))
```

?

Reply:
Also couldn't this use the `FromGlib` impl here instead?

Reply:
The idea was to not implement `FromGlib<u32>` for `char` since this could fail :)

Reply:
Ok!